tutorial/source/tensor_shapes.ipynb

```diff
@@ -528,22 +524,22 @@
     "\n",
     "# Test with enumeration.\n",
     "enumerated = True\n",
-    "test_model(model4, guide4, TraceEnum_ELBO(max_iarange_nesting=2))"
+    "test_model(model4, guide4, TraceEnum_ELBO(max_plate_nesting=2))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Automatic broadcasting via broadcast poutine<a class=\"anchor\" id=\"Automatic-broadcasting-via-broadcast-poutine\"></a>\n",
+    "### Automatic broadcasting inside pyro.plate<a class=\"anchor\" id=\"Automatic-broadcasting-inside-pyro-plate\"></a>\n",
```

Comment:
I think while this section is good as an appendix, it probably needs to be reworded slightly (maybe minor tweaks to the example too) in light of recent changes. I am happy to do so in a separate PR, and incorporate any suggestions you have.

Reply:
SGTM. I've reworded a bit, and I think further changes can be pushed to a follow-up PR.